types/yeoman-test/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for yeoman-test 4.0
+// Type definitions for yeoman-test 5.0
```

Comment:
All changes in this PR exist in version `4.0.2` as well.

Please change the version number to `4.0`.